### ISSUE TITLE
Set different labels above left and right months in DateRangePicker

### ISSUE THREAD

**Issue by DimaBur** [airbnb/react-dates#1842]
Is it possible to set content depends on "start" and "end" months?
I tried to put it in `renderMonthElement` but it's props the same for start and end months, and I can't write a condition to render different labels. 
Need to set labels `from` and `till` like on example image:
![Снимок экрана 2019-10-17 в 19 01 38](https://user-images.githubusercontent.com/13535011/67027462-44f31f00-f112-11e9-9a9f-38194fe3eb91.png)

**Comment by Sa-meer** [comment]
Well, there are some props provided to add something like that :  
`CalendarInfoPositionShape & renderCalendarInfo `  [check this](https://airbnb.io/react-dates/?path=/story/drp-calendar-props--with-info-panel-default) and by adding some CSS you can achieve this I guess.

**Comment by DimaBur** [comment]
Yes I know about this props, but they receive the same arguments for start and end months, so it's not possible to distinguish them. I solved this problem only by forking and customize this repo.

**Comment by jyotsana-khaparde** [comment]
Hello, 
I was using **DateRangePicker** and i wanted to show labels for the start date and end date for example -> 
![image](https://user-images.githubusercontent.com/58733142/162171478-ab818c6b-db0a-4844-9f87-a5a585343cff.png)

which props can be used for it? 
for placeholder i have used **startDatePlaceholderText, endDatePlaceholderText** props which is working.
for labels i have tried **startDateTitleText, endDateTitleText also startDateAriaLabel, endDateAriaLabel**  props but it's not working.

Does anyone have an idea how to achieve this? any help will be appreciated :-)
Thanks!!

**Comment by ljharb** [comment]
@jyotsana-khaparde please don't post the same content in multiple places; your comment is a duplicate of #2191.
